src/components/ResourceCard/EditResource/index.js

```diff
@@ -26,7 +26,7 @@ const EditResource = (props) => {
 
   useEffect(() => {
     showResourceDescribeActivity(resource, match.params.activityId);
-  }, []);
+  }, [match.params.activityId, resource, showResourceDescribeActivity]);
```

Comment:
I think I removed resource here
Did you add it?

Reply:
no i didn't, it should not have any dependency

Reply:
It should have those but not resource

Reply:
I will update it